.travis.yml

```diff
@@ -19,7 +19,7 @@ script:
   # check for changelog ReST compliance
   - rst2html.py --halt=warning changelog.rst >/dev/null
   # check for pep8 errors, only looking at branch vs master. If there are errors, show diff and return an error code.
-  - pep8radius master --docformatter --error-status || ( pep8radius master --docformatter --diff; false )
+  - pep8radius master --docformatter --diff --error-status
```

Comment:
@j-bennet this is what `--diff --error-status` do. `--diff` outputs the suggestion while `--error-status` abort test script.

These too options are composable, no need te run pep8radius twice.

@j-bennet i'm going to fake a error just for the showcase.